tests/Doctrine/Tests/ORM/Functional/ExtraLazyCollectionTest.php

```diff
@@ -39,6 +39,10 @@ public function setUp()
         $class->associationMappings['phonenumbers']['fetch'] = ClassMetadataInfo::FETCH_EXTRA_LAZY;
         $class->associationMappings['phonenumbers']['indexBy'] = 'phonenumber';
 
+        unset($class->associationMappings['phonenumbers']['cache']);
+        unset($class->associationMappings['articles']['cache']);
+        unset($class->associationMappings['users']['cache']);
```

Comment:
Yes, I get the point, but I'm unable to find where 
`$class->associationMappings['phonenumbers']['cache']` is setted to true...

I think that is some kind of "bug" into current test suite... looking at `CmsUser` it is not cacheable at all...